test/test_other.py

```diff
@@ -3411,6 +3411,15 @@ def test_jspi_add_function(self):
       '-Wno-experimental']
     self.do_runf('other/test_jspi_add_function.c', 'done')
 
+  @requires_jspi
+  def test_jspi_dylink(self):
+      self.run_process([
+        EMCC,
+        '-o', 'side.so',
+        test_file('other/test_jspi_dylink_side.c'),
+        '-sSIDE_MODULE'])
+      self.do_run_in_out_file_test('other/test_jspi_dylink_main.c', emcc_args=['side.so', '-sMAIN_MODULE=2', '-sJSPI'])
```

Comment:
We do already have `test_dlfcn_asyncify` in `test_core.py`  perhaps this new test would make sense alongside that one?

Reply:
For reference we do also have `with_asyncify_and_jspi` in test_core.py

Reply:
I can move it next to `test_dlfcn_asyncify`.